### PR TITLE
Documented the Symfony Messenger integration of Contao 5.1

### PR DESCRIPTION
**Pull request by Toflar**

### CODE REVIEW

File: docs/dev/framework/async-messaging.md [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+        {
+            foreach ($message->fileIds as $fileId) {
+                // Create your zip file asynchronously which can take a long while now 🔥
+            }
+        }
+    }
+    ```
+4. Dispatch the message from e.g. your controller and watch as the magic unfolds.
+5. Done! 🎉
+
```
Comment (leofeyer) [inline docs/dev/framework/async-messaging.md:265]: You could add something like this here:

_For a working example, take a look at the `SearchIndexMessage`, `SearchIndexMessageHandler` and `SearchIndexListener` classes to see how Contao uses the messenger to create and update the search index._

But feel free not to. 😄

Comment (Toflar) [reply docs/dev/framework/async-messaging.md:265]: Sure, great idea! Added in https://github.com/contao/docs/pull/1228/commits/227ce0eabbf5c77d3298be27f8ec4c8fdfbece35

File: docs/dev/framework/async-messaging.md [hunk 2]
```diff
@@ -0,0 +1,304 @@
...
+    }
+    ```
+3. Create your message handler:
+    ```php
+    namespace App\Messenger;
+        
+    use Symfony\Component\Messenger\Attribute\AsMessageHandler;
+
+    #[AsMessageHandler]
+    class SearchIndexMessageHandler
```
Comment (fritzmg) [inline docs/dev/framework/async-messaging.md]:
```suggestion
    class CreateAsyncZipFileMessageHandler
```

That was likely the intention, I assume? :)

**Review by fritzmg — CHANGES_REQUESTED** [review]

File: docs/dev/framework/async-messaging.md (hunk 2 above)
Comment (Toflar) [reply docs/dev/framework/async-messaging.md]: Of course🙈

**Review by fritzmg — APPROVED** [review]

**Comment by fritzmg** [comment]
Thank you @Toflar  :)

**Pull request merged**